### PR TITLE
Created Canceled EPs page

### PR DESCRIPTION
**Pull request by slj**
Connects #2030

### CODE REVIEW

File: spec/models/establish_claim_spec.rb [hunk 1]
```diff
@@ -518,4 +518,19 @@
       end
     end
   end
+
+  context "#past_weeks" do
+    before do
+      establish_claim
```
Comment (joofsh) [inline spec/models/establish_claim_spec.rb]: minor nit: I don't think we need this line?

File: spec/models/establish_claim_spec.rb [hunk 2]
```diff
@@ -518,4 +518,19 @@
       end
     end
   end
+
+  context "#past_weeks" do
```
Comment (joofsh) [inline spec/models/establish_claim_spec.rb]: nice unit test 👍

File: spec/models/establish_claim_spec.rb (hunk 1 above)
Comment (slj) [inline spec/models/establish_claim_spec.rb]: You're right.  Thx for catching that.

File: spec/models/establish_claim_spec.rb (hunk 2 above)
Comment (slj) [inline spec/models/establish_claim_spec.rb]: Thx!

**Pull request merged**